src/output_mqtt.c

```diff
@@ -511,11 +523,12 @@ struct data_output *data_output_mqtt_create(struct mg_mgr *mgr, char *param, cha
     char const *path_devices = "devices[/type][/model][/subtype][/channel][/id]";
     char const *path_events = "events";
     char const *path_states = "states";
+    char const *path_online = "online";
```

Comment:
I like the default payloads of online / offline.

I think it is confusing to have the default path be `online` with a potential value of `offline`. It feels like having a topic of `true` with potential values of `true` or `false`.

In HA's full configuration example the path is `availability`: https://www.home-assistant.io/integrations/binary_sensor.mqtt/#full-configuration-with-json-data

Could we do the same?

Reply:
That seems reasonable to me.

I personally prefer a topic of online and values of ON/OFF so that it is a binary sensor, which can then also be used for availability, but I realize that this is a non-standard approach, and that it is me taking the position that the non-alignmentof binary sensors and availability in HA is a bug.